flask_admin/contrib/sqla/ajax.py

```diff
@@ -58,16 +58,13 @@ def format(self, model):
 
         return getattr(model, self.pk), as_unicode(model)
 
-    def get_query(self):
```

Comment:
This is incompatible change. `get_query` allowed subclassing and mutating the generic query object in child admin classes.